inlong-audit/conf/application.properties

```diff
@@ -16,15 +16,17 @@
 # specific language governing permissions and limitations
 # under the License.
 #
-# proxy.type: pulsar / tube / kafka
+# the MQ type for audit proxy: pulsar / kafka
 audit.config.proxy.type=pulsar
 
 # Supports common JDBC protocol
 audit.config.store.mode=jdbc
 
 # manger config
 manager.hosts=127.0.0.1:8083
-proxy.cluster.tag=default_cluster
+
+# Get Kafka or Pulsar address from the cluster tag.
```

Comment:
Plz, describe this configuration directly.

Reply:
Done